### PR TITLE
Configure whitesource for `compass-manager`

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: prow/jobs/scans/whitesource-periodics.yaml [hunk 1]
```diff
@@ -667,6 +667,57 @@ periodics: # runs on schedule
...
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "compass-manager-whitesource-scan"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-gc-project-env: "true"
+        preset-kms-gc-project-env: "true"
+        preset-kyma-encryption-key: "true"
+        preset-kyma-keyring: "true"
+        preset-whitesource-product-kyma: "true"
+        preset-wssagent-keys: "true"
+        preset-sa-gke-kyma-integration-kyma-project-whitesource: "true"
```
Comment (Ressetkk) [inline prow/jobs/scans/whitesource-periodics.yaml:684]: I don't think you need encryption keys presets here. Please review if you need this much presets in this job.

I know it's a little bit out of scope but if it's new repo let's try to keep it as tidy as possible.

Comment (Ressetkk) [reply prow/jobs/scans/whitesource-periodics.yaml]: Im not sure. @dekiel can you propose something? I believe this number of presets is way too much for this job to work.

Comment (dekiel) [reply prow/jobs/scans/whitesource-periodics.yaml]: You can inherit different configSet which has less presets if it exist. If there is no configSet which suits your needs, you can crate new one. If this is something specific for the data file, use a local configSet. If you expect to use this configSet in other files go for global configSet. If this is something specific for one job, define values in jobConfig directly. Generating from templates doesn't force you to use any preset. Which preset your prowjob will use depends on which configSets you inherit and which presets your jobConfig define.

Comment (dekiel) [reply prow/jobs/scans/whitesource-periodics.yaml]: @VOID404 [here](https://github.com/kyma-project/test-infra/blob/main/development/tools/cmd/rendertemplates/README.md) you'll find docs with explanation for configSet term. Global configSets are defined [here](https://github.com/kyma-project/test-infra/blob/ccc26237b3526dd0785960b497c6d38a9b953203/templates/config.yaml#L75).

File: prow/jobs/scans/whitesource-periodics.yaml [hunk 2]
```diff
@@ -667,6 +667,56 @@ periodics: # runs on schedule
...
+      annotations:
+        pipeline.trigger: "periodic"
+        testgrid-create-test-group: "false"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "compass-manager-whitesource-scan"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-gc-project-env: "true"
+        preset-kms-gc-project-env: "true"
+        preset-kyma-keyring: "true"
```
Comment (Ressetkk) [inline prow/jobs/scans/whitesource-periodics.yaml:680]: Those 3 presets are not sed by whitesource script. Please remove them

**Review by Ressetkk — APPROVED** [review]

**Pull request merged**